### PR TITLE
Added starter packs

### PR DESCRIPTION
**Pull request by ellenk7**

### CODE REVIEW

File: src/assets/data/starter-pack-info.js [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+              all HackUCI 2021 attendees with 3 months free access to their
+              “Hacker” plan, which includes private projects, 10x the storage,
+              4x the memory, 4x the speed, and more. Sign up for your account{" "}
+              <a href="https://repl.it/login?goto=/claim?code=HackUCI2021">
+                here
+              </a>
+              !
+            </p>
+          </>
+        )
```
Comment (ChaseC99) [inline src/assets/data/starter-pack-info.js:293]: Remove
```suggestion
```

File: src/assets/data/starter-pack-info.js [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+                Part One
+              </a>
+            </p>
+            <p>
+              <a href="https://www.loom.com/share/1980b9cfcbfc459498bb0f73ac0bbda1">
+                Part Two
+              </a>
+            </p>
+          </>
+        )
```
Comment (ChaseC99) [inline src/assets/data/starter-pack-info.js:197]: Remove
```suggestion
```

File: src/assets/data/starter-pack-info.js [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+            <p>
+              Interested in Wolfram mentorships or internships? Check out our
+              opportunities here:{" "}
+              <a href="https://www.wolfram.com/company/careers">
+                https://www.wolfram.com/company/careers
+              </a>
+            </p>
+            <p>Good luck, and happy hacking!</p>
+          </>
+        )
```
Comment (ChaseC99) [inline src/assets/data/starter-pack-info.js:162]: Remove
```suggestion
```

**Review by ChaseC99 — CHANGES_REQUESTED** [review]
[#vh-starter-packs](https://venushacks.slack.com/archives/C01MMJYNXGR) is still finalizing the list in [this document](https://docs.google.com/document/d/12DfpPusC9yDzm-tYVI4EJYEsLm7OKBpLV6Atx13KKek/edit?usp=sharing). 

We should wait until that's done to deploy this.

File: src/assets/data/starter-pack-info.js (hunk 1 above)
Comment (ellenk7) [reply src/assets/data/starter-pack-info.js:293]: Yepp and also the live tmr by Mara Hart!

**Pull request closed without merging**